src/main/java/hudson/model/ExternalJob.java

```diff
@@ -122,6 +122,39 @@ public String getDisplayName() {
             return Messages.ExternalJob_displayName();
         }
 
+        /**
+         * Needed if it wants External Monitor Jobs are categorized in Jenkins 2.x.
+         *
+         * TODO: Override when the baseline is upgraded to 2.x
+         *
+         * @return A string it represents a ItemCategory identifier.
+         */
+        public String getCategoryId() {
+            return "standaloneprojects";
```

Comment:
I would expect it to be available via Enum or a static variable from the Jenkins core 2.0. IMHO there should be a TODO for it